examples/app-npm-webpack/messages/ru.json

```diff
@@ -0,0 +1,23 @@
+{
+	"ru": {
+		"intro-1": "Используйте Globalize для интернационализиции вашего приложения.",
+		"number-label": "Независимое число",
```

Comment:
@rxaviers, we discussed a better translation for the word "Standalone" (in Russian it looks strange to say "standalone" in this context) and decided to drop it keeping only translation for "Number", "Currency" etc.